tests/by-util/test_env.rs

```diff
@@ -55,6 +56,85 @@ fn test_if_windows_batch_files_can_be_executed() {
     assert!(result.stdout_str().contains("Hello Windows World!"));
 }
 
+#[test]
+fn test_debug_1() {
+    let ts = TestScenario::new(util_name!());
+    let result = ts
+        .ucmd()
+        .arg("-v")
+        .arg(&ts.bin_path)
+        .args(&["sleep", "0.1"])
```

Comment:
Is it possible to use something like `true` instead, or perhaps `echo Hello`? Call me silly, but all those sleeps add up.

Reply:
done - thanks.